fictadvisor-api/src/v2/utils/documentation/telegramGroup/getTelegramGroup.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiDocumentationParams } from '../decorators';
+import { TelegramGroupsByTelegramIdResponse } from '@fictadvisor/utils/responses';
+import { DefaultUnauthorizedResponse } from '../defaultResponses';
+
+export const TelegramGroupDocumentationGetTgGroup: ApiDocumentationParams = { 
+  isAuth: true,
```

Comment:
name of the documentation object must match such format: `{EntityName}Documentation{MethodName}`